db/schema.sql

```diff
@@ -0,0 +1,41 @@
+-- comments schema
+CREATE TABLE comment (
+  id SERIAL PRIMARY KEY,
+  Text TEXT NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  user_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
+  post_id INTEGER REFERENCES posts(id) ON DELETE CASCADE
+);
+--user schema
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  Name VARCHAR(255) UNIQUE NOT NULL,
+  Photo VARCHAR(255) NOT NULL,
+  Bio TEXT NOT NULL,
+  PostsCounter INTEGER NOT NULL DEFAULT 0,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+  foreign key (id) references posts(user_id) on delete cascade,
+    foreign key (id) references comment(user_id) on delete cascade, 
+    foreign key (id) references likes(user_id) on delete cascade
+);
+--post schema
+CREATE TABLE posts (
+  iAuthorId SERIAL PRIMARY KEY,
+  title VARCHAR(255) NOT NULL,
+  Text TEXT NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  commentsCounter INTEGER NOT NULL DEFAULT 0,
+    likesCounter INTEGER NOT NULL DEFAULT 0,
+    foreign key (id) references comment(post_id) on delete cascade,
+    foreign key (id) references likes(post_id) on delete cascade
+);
+--like shema
+CREATE TABLE likes (
+  post_id SERIAL PRIMARY KEY,
+  user_id SERIAL PRIMARY KEY,
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_2.md#:~:text=your%20project%20schema.-,Create%20and%20run%20the%20necessary%20migration%20files.,-Table%20and%20column), Please create all necessary migration files here in `./db/migrate` regarding tables name, foreign keys, and columns included on the tables.
[Hint]: You suppose to use `rails generate migration` command for that, See this [link](https://guides.rubyonrails.org/active_record_migrations.html#creating-a-migration) for more info.